modules/sdk/src/test/scala/org/tessellation/sdk/infrastructure/trust/TrustRatingCsvLoaderSuite.scala

```diff
@@ -0,0 +1,19 @@
+package org.tessellation.sdk.infrastructure.trust
+
+import cats.effect.IO
+
+import fs2.io.file.Path
+import weaver._
+import weaver.scalacheck._
+
+object TrustRatingCsvLoaderSuite extends SimpleIOSuite with Checkers {
+  test("load trust ratings from csv") {
+    val testFileLocation = getClass.getResource("/ratings.sample.csv").getPath
+    val inFile = Path(testFileLocation)
+
+    for {
+      csv <- TrustRatingCsvLoader.make[IO].load(inFile)
```

Comment:
for comprehension is not needed for a single element case.

it corresponds to that:
```scala
TrustRatingCsvLoader.make[IO].load(inFile).map { csv =>
  println(csv)
  success
}
```
or without println to that:
```scala
TrustRatingCsvLoader.make[IO].load(inFile).as(success)
```

Reply:
Changed.